Description: fix pkg-config files:
 Upstream's OpenIPMIpthread.pc.in file passes "-pthread" on
 CFLAGS, but not on LDFLAGS.  Even though this might be OK, GCC
 recommends that, when "-pthread" is used, it should be passed on
 CFLAGS and LDFLAGS.  This patch does that.
Author: Simon McVittie <smcv@debian.org>
Reviewed-By: Sergio Durigan Junior <sergio.durigan@canonical.com>
Bug-Debian: https://bugs.debian.org/474087
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1311888
Forwarded: https://sourceforge.net/p/openipmi/code/merge-requests/6/

Index: b/OpenIPMIpthread.pc.in
===================================================================
--- a/OpenIPMIpthread.pc.in
+++ b/OpenIPMIpthread.pc.in
@@ -7,5 +7,5 @@ Name: OpenIPMIpthread
 Description: Pthread OS handler for OpenIPMI
 Version: @VERSION@
 Requires: OpenIPMI
-Libs: -L${libdir} -lOpenIPMIutils -lOpenIPMIpthread
+Libs: -L${libdir} -lOpenIPMIutils -lOpenIPMIpthread -pthread
 Cflags: -I${includedir} -pthread
